apps/omg_eth/lib/eth.ex

```diff
@@ -55,30 +55,8 @@ defmodule OMG.Eth do
   @spec syncing?() :: boolean
   def syncing?, do: node_ready() != :ok
 
-  @doc """
-  Send transaction to be singed by a key managed by Ethereum node, geth or parity.
-  For geth, account must be unlocked externally.
-  If using parity, account passphrase must be provided directly or via config.
-  """
-  @spec send_transaction(map(), send_transaction_opts()) :: {:ok, hash()} | {:error, any()}
-  def send_transaction(txmap, opts \\ []) do
-    case backend() do
-      :geth ->
-        with {:ok, receipt_enc} <- Ethereumex.HttpClient.eth_send_transaction(txmap), do: {:ok, from_hex(receipt_enc)}
-
-      :parity ->
-        with {:ok, passphrase} <- get_signer_passphrase(txmap.from),
-             opts = Keyword.merge([passphrase: passphrase], opts),
-             params = [txmap, Keyword.get(opts, :passphrase, "")],
-             {:ok, receipt_enc} <- Ethereumex.HttpClient.request("personal_sendTransaction", params, []) do
-          {:ok, from_hex(receipt_enc)}
-        end
-    end
-  end
-
-  def backend do
-    Application.fetch_env!(:omg_eth, :eth_node)
-    |> String.to_existing_atom()
+  def backend() do
```

Comment:
Is this needed? Looks like not being used.

Reply:
Indeed it's only used in tests. will remove!